py/range.go

```diff
@@ -104,6 +106,42 @@ func (r *Range) M__iter__() (Object, error) {
 	}, nil
 }
 
+func (r *Range) M__str__() (Object, error) {
+	return r.M__repr__()
+}
+
+func (r *Range) repr() (Object, error) {
+	var out bytes.Buffer
```

Comment:
consider using [strings.Builder](https://godoc.org/strings#Builder) instead? (needs Go >= 1.10, though)

Reply:
strings.Builder is more recommended way :)

Reply:
I modified the code. Would you please review it again?